core/src/test/java/io/questdb/test/cutlass/line/tcp/LineTcpReceiverTest.java

```diff
@@ -745,6 +746,7 @@ public int openRW(LPSZ name, long opts) {
                                 --count == 0
                 ) {
                     renameTable(weather, meteorology);
+                    Os.sleep(100);
```

Comment:
Sleeps in tests are usually a hack. What is this actually waiting for?

Reply:
debug debris, removed